spec/index_spec.rb

```diff
@@ -76,7 +76,7 @@
   context "#[]" do
     before do
       @id = Daru::Index.new [:one, :two, :three, :four, :five, :six, :seven]
-      @mixed_id = Daru::Index.new ['a','b','c',:d,:a,0,3,5]
+      @mixed_id = Daru::Index.new ['a','b','c',:d,:a,8,3,5]
```

Comment:
The `0` was a key as well as index value, and since the first input that is `0` was a key the new code treated the whole input as key values rather than index values during [this test](https://github.com/v0dro/daru/pull/54/files#diff-4dd64e5576c9862a56edb0dbbab51831L96), that's why I had to change it to `8` so that the code treats input as index values.